touchforms/backend/xformserver.py

```diff
@@ -70,22 +70,20 @@ def do_POST(self):
             data_out = handle_request(data_in, self.server)
             reply = json.dumps(data_out)
         except (Exception, java.lang.Exception), e:
-            msg = ""
-            error_type = None
+            msg = ''
             if isinstance(e, java.lang.Exception):
                 e.printStackTrace()  # todo: log the java stacktrace
             elif isinstance(e, urllib2.HTTPError):
                 if e.headers.get("content-type", "") == "text/plain":
                     msg = e.read()
-            elif isinstance(e, TouchFormsException):
-                error_type = type(e).__name__
 
-            logging.exception('error handling request')
+            info = sys.exc_info()
```

Comment:
Sounds legit. I have no idea how one normally handles java exceptions in Jython.